renpy/common/00gui.rpy

```diff
@@ -125,17 +128,12 @@ init -1150 python in gui:
             if renpy.variant(variant):
                 f()
 
-    def rebuild():
-        """
-        :doc: gui
-
-        Rebuilds the GUI.
-
-        Note: This is a very slow function.
-        """
+        for i in config.translate_clean_stores:
+            renpy.python.clean_store_backup.backup_one("store." + i)
```

Comment:
This was there previously, but feels kind of unprincipled to me. Specifically, it's not clear why we want to run this here, but not when the language is changed.

Reply:
Yeah, I'm not a fan either, but best I can determine it is needed to retain and carry updated values through the main menu/end-of-game cycle. Without it, values in the `gui` store reset to whatever the were when the game originally launched, even if they've been updated since, and it's possible to end up in a situation where the values visible in `gui` do not match the styles being seen (even though the styles are correct the `gui` values are lagging behind).